discord/enums.py

```diff
@@ -616,13 +616,15 @@ class StickerFormatType(Enum):
     png = 1
     apng = 2
     lottie = 3
+    gif = 4
 
     @property
     def file_extension(self) -> str:
         lookup: dict[StickerFormatType, str] = {
             StickerFormatType.png: "png",
             StickerFormatType.apng: "png",
             StickerFormatType.lottie: "json",
+            StickerFormatType.gif: "gif",
         }
         return lookup[self]
```

Comment:
I think to avoid potential `KeyError`s in the future, we should use `get` and default the type.
```suggestion
        return lookup.get(self, "png")
```

Reply:
How would this result in a KeyError? All of the able to get values are in the enum.

Reply:
Also nevermind, seemingly underestimated enums: https://discord.com/channels/881207955029110855/881735314987708456/1073599803214282784

Reply:
The reason this PR is being made in the first place is because Discord introduced the gif type which resulted in a KeyError, hence if they were to theoretically introduce another type we would run into the same issue.

Reply:
I'm unsure if this is the best solution, so I added a TODO note here that will remind us to come back and improve this. For now though, let's just get a patch released ASAP.
```suggestion
        # TODO: Improve handling of unknown sticker format types if possible
        return lookup.get(self, "png")
```